### DISCUSSION TITLE
Can I buy Next.js merch?

### DISCUSSION THREAD

**Discussion by filipesmedeiros** [vercel/next.js#30346] [Help] [answered]
hey! where can i buy those awesome items that were on the keynote background? namely the bug black triangle and the coffe mug with the next (vercel?) logo?

thanks!

**Comment by Aman-Kaushik13** [comment]
I think @leerob could answer the question, as I had the same question when one of my friends got some merch

**Comment by SuttonJack** [comment, marked as answer]
Mug: https://edge-mug.vercel.app/edge

Some of the talks had "sample" commerce projects with real URLs in them 🙂

**Comment by jarrisondev** [comment]
Vercel drop his merch page? i'm searching the page for buy the cup but i cant find it

**Reply by leerob** [reply]
The mug linked above is currently sold out. We do periodic giveaways of merch on Twitter and at events.

**Reply by m1heng** [reply]
😭 please add more inventory, really wish to have one.

**Reply by luissquall** [reply]
Yes please! I would love to buy a couple of those beautiful Vercel t-shirts.

**Comment by juandadev** [comment]
How can we be aware of new merch supply?

**Reply by leerob** [reply]
We'll be doing some swag giveaways on Twitter in the future!

https://twitter.com/vercel
